### PR TITLE
fix nits incorrect udpate into correct update

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

**What this PR does / why we need it**:
fix nits incorrect udpate into correct update

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by cblecker** [comment]
/assign @smarterclayton
for approval

**Comment by allencloud** [comment]
friendly ping @smarterclayton ☀️  :lion:

**Comment by allencloud** [comment]
friendly ping @smarterclayton ☀️  :lion:

**Comment by allencloud** [comment]
It is good to see all tests green.
And could someone prove this via `/approve no-issue` ? 😃

**Comment by allencloud** [comment]
Any update? 🐯 :lion:

**Comment by cblecker** [comment]
@smarterclayton Should this have `/approve no-issue` as it's a typo fix?

**Comment by allencloud** [comment]
ping @smarterclayton 😄

**Comment by allencloud** [comment]
ping @smarterclayton 😄

**Comment by allencloud** [comment]
@smarterclayton Should this have /approve no-issue as it's a typo fix?

File: pkg/apis/apps/validation/validation_test.go [hunk 1]
```diff
@@ -249,7 +249,7 @@ func TestValidateStatefulSet(t *testing.T) {
 				UpdateStrategy: apps.StatefulSetUpdateStrategy{Type: apps.RollingUpdateStatefulSetStrategyType},
 			},
 		},
-		"invalid udpate strategy": {
+		"invalid update strategy": {
```
Comment (nikhita) [inline pkg/apis/apps/validation/validation_test.go]: This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464. Closing.

/close

**Pull request closed without merging**